CHANGES.rst

```diff
@@ -0,0 +1,13 @@
+Changes
+=======
+
+This is a record of all past ``gettsim`` releases and what went into them in reverse
+chronological order. We follow `semantic versioning <https://semver.org/>`_ and all
+releases are available on `Anaconda.org <https://anaconda.org/gettsim/gettsim>`_.
+
+0.0.1 - 2019-
```

Comment:
I think someone started with 0.1, am I right?

Reply:
Who was it :).

Reply:
Maybe one should not deal with stuff like that after dealing with two kids? :-)